### DISCUSSION TITLE
Custom catalog for asteroid flyby

### DISCUSSION THREAD

**Discussion by vossinakis** [Stellarium/stellarium#3107] [Q&A] [answered]
Hi, how can I add a custom catalog to stellarium with ra and dec positions generated by NASA's horizons?? I know this is possible in cartes du ciel. Adding catalogs this way is more accurate during close flybys because of Earth's perturbations.

**Comment by gzotti** [comment, marked as answer]
It is not possible. Use cartes du ciel for this.
